recipes/pygyat/recipe.yaml

```diff
@@ -0,0 +1,52 @@
+context:
+  name: pygyat
+  version: "1.0.6b0"
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
there is 1.0.7 out anyway now so i'll just use that